.codesandbox/ci.json

```diff
@@ -0,0 +1,41 @@
+{
+  "buildCommand": "codesandbox-ci",
+  "packages": [
+    "bundles/pixi.js",
+    "bundles/pixi.js-webworker",
+    "packages/accessibility",
+    "packages/app",
+    "packages/assets",
+    "packages/compressed-textures",
+    "packages/core",
+    "packages/display",
+    "packages/events",
+    "packages/extensions",
+    "packages/extract",
+    "packages/filter-alpha",
+    "packages/filter-blur",
+    "packages/filter-color-matrix",
+    "packages/filter-displacement",
+    "packages/filter-fxaa",
+    "packages/filter-noise",
+    "packages/graphics",
+    "packages/graphics-extras",
+    "packages/math-extras",
+    "packages/mesh",
+    "packages/mesh-extras",
+    "packages/mixin-cache-as-bitmap",
+    "packages/mixin-get-child-by-name",
+    "packages/mixin-get-global-position",
+    "packages/particle-container",
+    "packages/prepare",
+    "packages/sprite",
+    "packages/sprite-animated",
+    "packages/sprite-tiling",
+    "packages/spritesheet",
+    "packages/text",
+    "packages/text-bitmap",
+    "packages/text-html"
```

Comment:
Nice, I actually just pushed this change as well 😄 

I was initially trying to avoid spamming the CodeSandbox CI registry with too many modules, but I don't think it will make a difference